src/geo-brand-presence/guidance-geo-brand-presence-handler.js

```diff
@@ -13,23 +13,34 @@
 import { notFound, ok } from '@adobe/spacecat-shared-http-utils';
 import { convertToOpportunityEntity } from './opportunity-data-mapper.js';
 
-function getSuggestionValue(suggestions) {
-  // TODO: convert the suggestions to markdown table, read
-  // the url, type, keywords, copy from suggestion and add it to the table
-  // return the markdown table
-  let suggestionValue = '| Url | Questions | Screenshot |\n |-----|-----------|------------|\n';
-  suggestions.forEach((suggestion) => {
-    suggestionValue += `| ${suggestion.url} | ${suggestion.q.join('\n')} | [![${suggestion.name}](${suggestion.previewImage})](${suggestion.screenshotUrl})|\n`;
-  });
-  return suggestionValue;
+const POSSIBLE_SUBTYPES = ['guidance:geo-brand-presence', 'guidance:geo-faq'];
+
+function getSuggestionValue(suggestions, subType, log) {
+  if (subType === 'guidance:geo-faq') {
+    let suggestionValue = '| URL | Question | Answer | Sources |\n|-----|----------|-------|--------|\n';
+    suggestions.forEach((suggestion) => {
+      const sources = suggestion.sources ? suggestion.sources.map((source, sourceIndex) => `[${sourceIndex + 1}] ${source}`).join('<br>') : '';
+      suggestionValue += `| ${suggestion.page_url} | ${suggestion.question} | ${suggestion.answer} | ${sources} |\n`;
+    });
+    return suggestionValue;
+  } else if (subType === 'guidance:geo-brand-presence') {
+    let suggestionValue = '| Url | Questions | Screenshot |\n |-----|-----------|------------|\n';
+    suggestions.forEach((suggestion) => {
+      suggestionValue += `| ${suggestion.url} | ${suggestion.q.join('\n')} | [![${suggestion.name}](${suggestion.previewImage})](${suggestion.screenshotUrl})|\n`;
+    });
+    return suggestionValue;
+  } else {
+    log.warn(`Unsupported subType: ${subType}`);
+    return notFound();
+  }
 }
 
 export default async function handler(message, context) {
   const { log, dataAccess } = context;
   const { Audit, Opportunity, Suggestion } = dataAccess;
   const { auditId, siteId, data } = message;
   const { suggestions } = data;
-  log.info(`Message received in guidance:geo-brand-presence handler: ${JSON.stringify(message, null, 2)}`);
+  log.info(`Message received in guidance handler: ${JSON.stringify(message, null, 2)}`);
```

Comment:
I know it was like this, but this works well with coralogix and `console`:
```suggestion
  log.info('Message received in guidance handler:', message);
```